### ISSUE TITLE
Writing jsons for event and equipment post functionalities

### ISSUE THREAD

**Issue by GozdeUnver** [bounswe/2021SpringGroup5#262]
@salih997 and I will write the jsons for event and equipment post functionalities that will be used in the development. We will try to write the functionalities according to activity stream structure if they can be represented as an activity.

**Comment by GozdeUnver** [comment]
We wrote jsons for :

- see detail_2 Event (act stream format)
- see detail_2 Equipment (act stream format)
- changeEquipmentInfo()
- changeEventInfo()
- createEvent()
- createEquipmentPost()
- deleteEquipmentPost()
- getWaitingApplications()
- getAcceptedApplications()
- getRejectedApplications()
- getSpectators()
- getEventAnalytics()
- acceptApplicants()

Not all functionalities were completed because we have to discuss it with the frontend team on next Tuesday when we will have a group meeting.

**Comment by GozdeUnver** [comment]
In the meeting we decided that 
- Cancelevent method will be removed from the class diagram and deleteEvent will be added instead. Event status will be a string in the class diagram
- There is no accept applicants method but instead changeStatusRequest() will behave like it by changing the statuses of the applicants. getApplicants method is actually gathering all the applicants whose statuses are approved.
- When repeating frequency=2 it means the event will be repeated for two weeks on the same day automatically, this repeatition should be triggered by the backend code
Location requirement means if it is a special location such as "halısaha"
We didn't write any json together but we decided that from now on the whole team is going to decide on the next jsons that will be implemented instead of only the backend team. So new issue will be opened.
